gilbreth_perception/config/model_list.yaml

```diff
@@ -0,0 +1,15 @@
+part_list:
+ - name: disk 
+   path: /model/disk.pcd
+   pick_point_index: 137
+ - name: gear
+   path: /model/gear.pcd
+   pick_point_index: 40
+ - name: piston_rod
+   path: /model/piston_rod.pcd
+   pick_point_index: 48
+pick_orientation:
+ x: 0
+ y: 1
+ z: 0
+ w: 0
```

Comment:
I think you'd want to define a pose on a per object basis, meaning each object gets its own pick pose, an example of this would be:
```
- name: part_a
   path: models/part_a.pcd
   pick_pose: [0.2, 0.2, 0.2, 0.0, 0.0, 1.57]  # [px, py, pz, rx, ry, rz]
```
The pick pose can be set much more easily if you use euler angles ( just three variables rx, ry, rz) instead of a quaternion.

Reply:
Changed, now the yaml file is the same as your suggestion.